common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBArtifactStore.scala

```diff
@@ -476,10 +488,11 @@ class CosmosDBArtifactStore[DocumentAbstraction <: DocumentSerializer](protected
         (clusterId, clusterIdValue))
     val fieldsToRemove = Seq(_id, _rev)
     val mapped = transform(json, fieldsToAdd, fieldsToRemove)
+    val jsonString = mapped.compactPrint
     val doc = new Document(mapped.compactPrint)
```

Comment:
maybe should avoid calling `mapped.compactPrint` twice?

Reply:
Missed that while refactoring. Reused existing computed json